yt_dlp/extractor/ard.py

```diff
@@ -360,11 +362,32 @@ def _extract_episode_info(self, title):
     def _real_extract(self, url):
         display_id = self._match_id(url)
 
+        query = {
+            'embedded': 'false',
+            'mcV6': 'true'
+        }
+        headers = {}
+
+        if self._get_cookies(self._TOKEN_URL).get('ams'):
+            token = self._download_json(
+                self._TOKEN_URL, display_id, 'Fetching token for age verification',
+                'Unable to fetch age verification token', fatal=False)
+            id_token = traverse_obj(token, ('idToken', {str}))
+            user_id = traverse_obj(id_token, ({jwt_decode_hs256}, ('user_id', 'sub'), {str}), get_all=False)
+            if not id_token or not user_id:
+                self.report_warning('Unable to extract token, continuing without authentication')
+            else:
+                headers['x-authorization'] = f'Bearer {id_token}'
+                query['userId'] = user_id
+
+            age_rating = traverse_obj(id_token, ({jwt_decode_hs256}, 'age_rating'), get_all=False)
+            if age_rating != 18:
+                self.report_warning(f'Authenticated age_rating is not 18, but "{age_rating}", '
+                                    'video might still be blocked')
```

Comment:
We don't want to throw an additional warning if the token couldn't even be fetched.
I would also not print the age rating, as for unverified accounts it is `None`.
```suggestion
                if decoded_token.get('age_rating') != 18:
                    self.report_warning('Account is not verified as 18. Video might be unavailable.')
```